hw.js

```diff
@@ -1,25 +1,30 @@
-​// ------ FOR ALL THE EXCERCISES ONLY USE ARROW FUNCTIONS  ----- //
+​/* FOR ALL THE EXCERCISES ONLY USE ARROW FUNCTIONS   
 ​
 
+ PART 1:  USE MAP IN THE FOLLOWING EXCERCISES 
 
-// PART 1:  USE MAP IN THE FOLLOWING EXCERCISES //
+ A) write a function called swapCase that takes a string of words and uses .map and your newly written capitalize()
+ function to return a string where every other word is in all caps.
+ Hint: look up Array.prototype.map on MDN and see what arguments the .map callback can take.
+ ex: swapCase('hello stranger , what do we have today? ') // => " HELLO stranger , WHAT do WE have TODAY ?" */
 
-// A) write a function called swapCase that takes a string of words and uses .map and your newly written capitalize()
-// function to return a string where every other word is in all caps.
-// Hint: look up Array.prototype.map on MDN and see what arguments the .map callback can take.
-// ex: swapCase('hello stranger , what do we have today? ') // => " HELLO stranger , WHAT do WE have TODAY ?"
+
+//map
 var swapCase = function(string) {
-  // Codeeeee
-};
+  return (string.split(" ").map(letter => letter.toUpperCase())).join(" ")
```

Comment:
so we wanted every other word to be captalized , just a small fix to this line and your code works perfectly ! ```    .map((word, letter) => (letter % 2 === 0 ? word.toUpperCase() : word))```